Documentation/CameraSystem/WindowsMixedRealityCameraSettings.md

```diff
@@ -35,8 +35,20 @@ The Windows Mixed Reality Camera Settings also supports a profile. This profile
 
 With this setting on HoloLens 2, you can enable hologram alignment in your mixed reality captures. If enabled, the platform will provide an additional HolographicCamera to the app when a mixed reality capture photo or video is taken. This HolographicCamera provides view matrices corresponding to the photo/video camera location, and it provides projection matrices using the photo/video camera field of view. This will ensure that holograms, such as hand meshes, remain visibly aligned in the video output.
 
+### HoloLens 2 reprojection method
+
+Set the initial method for HoloLens 2 reprojection here. To update the reprojection method at runtime, access the `WindowsMixedRealityReprojectionUpdater` like so:
```

Comment:
Nit: Any thoughts on adding a blurb about why you might want to do this? (i.e. if you're experiencing some bad look at feel with the defaults, this lets you use some of the other reprojection techniques to see if they will improve your stuff)

Reply:
Added a bit more info about scenarios when you might want to try alternate methods